students/Kenneth_Murray/Lesson6/testcases/test_mailroom.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+"""
+Unit tests for the mailroom program
+"""
+
+
+contributor_list = [('Ken', 'Murray', '10.00', '4.00'),
```

Comment:
this should go in a `def setUp(self):` function which works basically the same as an `__init__`.  The vars and methods within will be called before the start of EACH test.  So if you change then in one test, they will reset again before the next.